### DISCUSSION TITLE
Just updated from AIR_SDK_33.1.1.795 to the latest 50.2.3 and things don't run in CS5.5 when targetting Air.

### DISCUSSION THREAD

**Discussion by jerryComo** [airsdk/Adobe-Runtime-Support#2791] [General]
No errors - but it just doesn't run. Whereas if I make a new Air project and put some basic code in it runs fine. Also the swf generated by my old project runs fine in the player and also works when I target FP 11 , but I'd rather be able to use it within the IDE and not have keep commenting out all the references to NativeApplication

I'm picking apart an old file to see if I can find what is causing the no-run situation.  It was fine with 33.1.1.795 - has there been some larger change since 50 ?

**Comment by ajwfrost** [comment]
There shouldn't be anything that causes such an incompatibility between those versions..  are you able to try mimicking the launch and use ADL from the command line, to see if that gives any other clues?

Other thoughts:
* could you create a small SWF that then just loads your application's SWF, and check for errors there? (if there's a problem with code definitions, you may then see a VerifyError being thrown)
* have you got ANEs? if so, can you remove those to see if they're the problem?
* not sure whether you'd get any output from Scout in this scenario but it's worth checking
* also worth checking the task manager to see if ADL is actually running, but perhaps it's just that the window isn't appearing (Scout would also show this scenario)

And I just thought .. are you using macOS? a relatively recent version of it? If so, there's every chance the the permissions/entitlements are causing a problem. We had this where Animate couldn't run ADL because ADL needed certain entitlements, but macOS needed Animate to have these entitlements instead. There is a workaround that can be done for this scenario, available at; https://github.com/airsdk/Adobe-Runtime-Support/issues/995#issuecomment-930924193

thanks
